react-component-template/index.jsx

```diff
@@ -12,19 +14,8 @@ export default class {{PascalName}} extends Component {
     name: '{{titleName}}'
   }
 
-  constructor (props) {
-    super(props)
-    // initialize state values
-    this.state = {}
-  }
-
-  // use the pure-render mixin without the mixin. This allows us to use es6
-  // classes and avoid "magic" code. NOTE: if this component is used directly
-  // by react-router, you should delete it, otherwise, the <Link> component will
-  // not cause a re-render
-  shouldComponentUpdate (...args) {
-    return shouldComponentUpdate.apply(this, args)
-  }
+  // initialize state values
+  state = {}
```

Comment:
i'm a little confused about whether or not @Jon-Biz supports this change or not.....

regardless, wasn't the `constructor` method and the `state` initialization added to act as a helpful guide for anybody utilizing these features? If `state` initialization is moved outside of the `constructor`, won't that be a problem if`state` needs to be initialized from `props` (as Jon outlined in his case list)?